nautobot/circuits/views.py

```diff
@@ -127,8 +129,23 @@ class CircuitUIViewSet(NautobotUIViewSet):
     queryset = Circuit.objects.all()
     serializer_class = serializers.CircuitSerializer
     table_class = tables.CircuitTable
-    # NOTE: This is how `NautobotUIViewSet` would define use_new_ui attr
-    # use_new_ui = ["list", "retrieve"]
+    object_detail_content = ObjectDetailContent(
+        detail_fields={
+            "Circuit": [
+                "status",
+                "provider",
+                "cid",
+                "circuit_type",
+                "tenant",
+                "install_date",
+                "commit_rate",
```

Comment:
Does it make sense to send directly as a django template variable, e.g. set this as `commit_rate|humanize_speed|placeholder`. I don't know if that is going overboard, but jut thinking out loud.

Reply:
The downside of that approach, I think, would be that we'd have to do multiple rendering passes on the template to get this to be interpreted as such, which would probably raise issues with escaping template content and so forth. I'll try and think about it a bit more though.